Pokemon Unity/Assets/Scripts/Save/CustomSaveEvent.cs

```diff
@@ -0,0 +1,27 @@
+﻿using UnityEngine;
+using System;
+
+[System.Serializable]
+public class CustomSaveEvent
+{
+    public DateTime EventTime;
+    public SaveEventType EventType;
+    public int SceneIndex;
+    public string ObjectName;
+    public SerializableVector3 ObjectPosition;
+
+    public CustomSaveEvent(DateTime eventTime, SaveEventType eventType, GameObject eventObject, int sceneIndex)
```

Comment:
Truth is, you dont need to pass a "DateTime" field when saving a game. Cause you're not saving in future or the past, so by default, it's always going to be => "DateTime.Now"